include/dumb.h

```diff
@@ -118,9 +127,9 @@ typedef long long dumb_off_t;
  */
 #if (__GNUC__ > 4 || (__GNUC__ == 4 && __GNUC_MINOR__ >= 6)) &&                \
     !defined __cplusplus
-_Static_assert(sizeof(dumb_off_t) >= 8, "fuse: off_t must be 64bit");
+_Static_assert(sizeof(dumb_off_t) >= 8, "dumb: off_t must be 64bit");
 #else
-struct dumb_off_t_needs_to_be_at_least_8_bytes {
+    struct dumb_off_t_needs_to_be_at_least_8_bytes {
```

Comment:
Remove the indentation? Conventionally, `#if`/`#else` don't introduce scope in this codebase.